src/main/java/org/apache/commons/imaging/formats/webp/chunks/WebPChunkVP8.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.imaging.formats.webp.chunks;
+
+import org.apache.commons.imaging.ImageReadException;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public final class WebPChunkVP8 extends WebPChunk {
+    private final int versionNumber;
+    private final int width;
+    private final int height;
+    private final int horizontalScale;
+    private final int verticalScale;
+
+    public WebPChunkVP8(int type, int size, byte[] bytes) throws ImageReadException {
+        super(type, size, bytes);
+
+        if (size < 10) {
+            throw new ImageReadException("Invalid VP8 chunk");
+        }
+
+        int b0 = bytes[0] & 0xFF;
+        // int b1 = bytes[1] & 0xFF;
+        // int b2 = bytes[2] & 0xFF;
```

Comment:
I like that these two are commented out, meaning that it was not a mistake to skip these. But we still need a comment, so other devs are aware of why this was done, please.